pkg/controller/aws/cache/claim.go

```diff
@@ -40,12 +40,17 @@ func AddClaim(mgr manager.Manager) error {
 	r := resource.NewClaimReconciler(mgr,
 		resource.ClaimKind(cachev1alpha1.RedisClusterGroupVersionKind),
 		resource.ManagedKind(v1alpha1.ReplicationGroupGroupVersionKind),
+		resource.WithManagedBinder(resource.NewAPIStatusManagedBinder(mgr.GetClient())),
+		resource.WithManagedFinalizer(resource.NewAPIStatusUnbindingManagedFinalizer(mgr.GetClient())),
 		resource.WithManagedConfigurators(
 			resource.ManagedConfiguratorFn(ConfigureReplicationGroup),
 			resource.NewObjectMetaConfigurator(mgr.GetScheme()),
 		))
 
-	name := strings.ToLower(fmt.Sprintf("%s.%s", cachev1alpha1.RedisClusterKind, controllerName))
+	name := strings.ToLower(fmt.Sprintf("%s.%s.%s",
```

Comment:
I like this pattern of explicitly defining `abstract` + `managed` + `api` as opposed to just appending the abstract kind to the controller name of the managed controller.